xapi-profiles-communication.md

```diff
@@ -48,14 +48,19 @@ implementing the algorithms.
 
 ## Authored Profiles
 
-ADL maintains a [centralized public repository of authored xAPI profiles](https://github.com/adlnet/xapi-authored-profiles) based on this specification. The repository of profiles are imported and synchronized regularly into ADL's Profile Server, [http://xapi.vocab.pub](http://xapi.vocab.pub). 
+ADL maintains a [centralized public repository of authored xAPI profiles](https://github.com/adlnet/xapi-authored-profiles) based on this specification. The repository of profiles are imported and synchronized regularly into ADL's Profile Server, [http://xapi.vocab.pub](http://xapi.vocab.pub).
 
 <a name="prof-server"></a>
 ## 1.0 Profile Server
 
 A Profile Server manages xAPI Profiles from a centralized location. [An RDF triple store](https://en.wikipedia.org/wiki/Triplestore) is responsible for the storage of Profiles.
 A Profile Server will allow administrators to add Profiles by their contents or by URI to the Profile Server
 
+When storing an xAPI Profile:
+* a Profile Server MUST NOT add any properties (not defined in this document) to the Profile
+* a Profile Server MAY add JSON-LD keywords (or aliases thereof) to a Profile anywhere they are legal in JSON-LD
+* a Profile Server MUST reject any Profiles that contain any properties not defined in this document (excluding correctly used JSON-LD keywords or aliases thereof)
```

Comment:
shouldn't we use *SHALL* and *SHALL NOT* to align with IEEE style ?

Reply:
Profile Server Version handling, what are the requirements for releasing a new version. Does the author of the profile manage / maintain the profile versions or is that solely a responsibility of the profile server.

-> Initial thoughts, profile server shouldn't be the only place that handles versioning, authors should be expected to include the version info in their profiles so that the version information is not lost on upload to a profile server.

-> Profile server may offer publish mechanics which applies publishes changes to a profile as a new version

There should / will be a Profile Server subgroup that gets into the details of what the Profile Server requirements are. The requirements added here were in relation to the additional properties requirement change.